app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/what-to-expect-in-virtual-coffee.mdx

```diff
@@ -0,0 +1,277 @@
+
+## Co-Working with Other Members
+
+<LeadText>
+
+Do you want to work or learn with people around, need help and a pair program, or only want to hang out with other members? Thanks to **Dan Ott**, we have a `#co-working-room` channel linked to our Zoom, so our members can connect anytime! It's open 24/7!
+
+</LeadText>
+
+### Joining Co-Working Session
+
+- Head over to the `#co-working-room` channel on Slack.
+- Click the green button to start a new session or join the ongoing session.
+
+Now you will be directed to our Zoom.
+
+</TextContainer>
+
+<TextContainer>
+
+## Virtual Coffee Events
+
+<LeadText>
+
+Our members voluntarily hold events from members to members, and you're most welcome to join!
+
+</LeadText>
+
+### Events List
+
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/coffee-table-groups">
+  	Coffee Table Groups
+  </Link>
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/lunch-and-learns">
+  	Lunch & Learns
+  </Link>
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/lightning-talk">
+  	Lightning Talk
+  </Link>
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/monthly-challenges">
+  	Monthly Challenges
+  </Link>
+- <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/hacktoberfest-initiative">
+  	Hacktoberfest Initiative
+  </Link>
+
+Every Monday, we drop the week's list of events in the `#announcement` channel on Slack. We also drop the schedule on the day of the events. There will be another announcement to join the events in the `#vc-events` channel 10 minutes before the events start, along with the link to join.
+
+You can find the complete schedule of the events on the <Link to="/events">Events</Link> page.
+
+### Joining Events
+
+- Head over to the `#vc-events` channel on Slack.
+- Click the "Join Event" button to join the event.
+
+Now you will be directed to the event's Zoom.
+
+</TextContainer>
+
+<TextContainer>
+
+## Virtual Coffee on Slack
+
+<LeadText>
+
+Much of Virtual Coffee occurs on Slack. We have channels in our Slack to accommodate our members, and we encourage you to participate in Slack to get the most out of Virtual Coffee!
+
+</LeadText>
+
+### Slack Channels
+
+See our <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/slack-channels-guide">Virtual Coffee Slack Channels Guide</Link> for the complete list of our Slack channels.
+
+And read our tips around Slack in the next section if you need to familiarize yourself with Slack.
+
+### Tips Around Slack
+
+#### ➡️ Using Tags
+
+You can tag members (e.g., `@name`) to notify them about your thread and draw their attention.
+
+But unless you are a Maintainer or granted permission by a Maintainer, don't use `@here` or `@channel`.
+Using these tags will notify everyone in the channel. It might be an afternoon at your time, but it is midnight in some parts of the world. And we want to respect all of our members.
```

Comment:
It might be an afternoon at your time, but that means it's midnight for some members. We want to respect all timezones.